packages/libs/client-utils/etc/client-utils-core.api.md

```diff
@@ -75,13 +75,13 @@ export const crossChainClient: <T = PactValue>(args_0: IClientConfig, args_1?: I
     data: ICommandResult;
     }, {
     event: "spv-proof";
-    data: {
+    data: Partial<{
     pactId: string;
     step: number;
     rollback: boolean;
     data?: Record<string, unknown> | undefined;
     proof?: string | null | undefined;
```

Comment:
If partial is used, we don't need to specify `?` or `|undefined`, correct?

Why is this partial? It's fairly sure what `spv-proof` should look like?

Reply:
I have addressed this